lang/javascript/javascript.talon

```diff
@@ -25,6 +26,7 @@ settings():
 
 (op | is) strict equal: " === "
 (op | is) strict not equal: " !== "
+op null or: " ?? "
```

Comment:
@david-tejada mentioned on slack that accuracy for "null or" hasn't been great. I've been using "null coal", which works _ok_. @david-tejada suggested "nullish" seems to work ok, tho the fact it's not a real word makes me a little nervous. Maybe "dub quest" (short for "double question mark")?

Reply:
How about `null else`?